packages/graph/hash_graph/lib/graph/src/datastore/postgres/mod.rs

```diff
@@ -131,12 +258,83 @@ mod tests {
     async fn get_entity_types() -> Result<(), sqlx::Error> {
         let pool = PostgresDatabase::new(&DB_INFO).await?.pool;
 
-        let _rows: Vec<EntityType> = sqlx::query_as("SELECT * from entity_types")
+        let _rows: Vec<EntityTypeRow> = sqlx::query_as("SELECT * from entity_types")
             .fetch_all(&pool)
             .await
             .report()
             .attach_printable("Could not select entity types")?;
 
         Ok(())
     }
+
+    #[ignore]
+    #[tokio::test]
+    async fn create_data_type() -> Result<(), sqlx::Error> {
+        let db = PostgresDatabase::new(&DB_INFO).await?;
+
+        let account_id = create_account_id(&db.pool).await?;
+
+        db.create_data_type(
+            DataType::new(serde_json::json!({"hello": "world"})),
+            account_id,
+        )
+        .await
+        .expect("Could not create data type");
```

Comment:
I think these `expect`s were added because the `PostgresDatabase::new` returns a `sqlx::Error` in the error case, which is not `DatastoreError` the rest returns. Should `PostgresDatabase::new` be changed, or do we just expect on that instead?

Reply:
There's a comment thread on the PostgresDatabase module where I suggested we should probably change the error types. However, in these tests we can just call `change_context(DatastoreError)`